lib/android-helpers.js

```diff
@@ -447,14 +447,15 @@ helpers.unlockWithUIAutomation = async function (driver, adb, unlockCapabilities
     throw new Error(`Invalid unlock type ${unlockType}`);
   }
   let unlockKey = unlockCapabilities.unlockKey;
-  if (!unlocker.isValidKey(unlockType, unlockKey)) {
+  if (!unlocker.isValidKey(unlockType, unlockKey) && unlockType !== SWIPE_UNLOCK) {
     throw new Error(`Missing unlockKey ${unlockKey} capability for unlockType ${unlockType}`);
   }
   const unlockMethod = {
```

Comment:
I know this is outside this PR strictly speaking, but it seems like this could be in the unlock helper, and there wouldn't need to be so much bleed into this helper. Something like
```js
import { unlock } from './unlock-helpers';

helpers.unlockWithUIAutomation = async function (driver, adb, caps) {
  return await unlock(driver, adb, caps.unlockType, caps.unlockKey);
};
```
Then the `unlock-helpers.js` would just need to expose for use (more for testing) something like
```js
async function unlock (adb, driver, type, key) {
  if (!helpers.isValidUnlockType(type)) {
    throw new Error(`Invalid unlock type '${type}'`);
  }
  if (!helpers.isValidKey(type, key)) {
    throw new Error(`Missing unlockKey '${key}' capability for unlockType '${type}'`);
  }
  const method = {
    PIN_UNLOCK: 'pinUnlock',
    PASSWORD_UNLOCK: 'passwordUnlock',
    PATTERN_UNLOCK: 'patternUnlock',
    FINGERPRINT_UNLOCK: 'fingerprintUnlock',
    SWIPE_UNLOCK: 'swipeUnlock',
  }[type];
  await helpers[method](adb, driver, type, key);
}

export{ unlock };
```